dataset_interfaces/gpt_generated.py

```diff
@@ -77,6 +77,7 @@ def generate_examples(self, num_examples) -> List[TestExample]:
                 expected_responses=expected_responses,
                 is_question=is_question,
                 uses_callback=self.uses_callback,
+                memory_span=self.memory_span,
```

Comment:
this could be set automatically in `TestExample.__post_init__` from the reference to the example generator, to minimize the probability of getting it wrong.